kmipclient/include/Logger.hpp

```diff
@@ -0,0 +1,19 @@
+//
+// Created by al on 09.04.25.
+//
+
+#ifndef LOGGER_HPP
```

Comment:
I would use library name as a prefix for an include guard names to avoid potential name clash with headers from other libraries.
```suggestion
#ifndef KMIP_CLIENT_LOGGER_HPP
```

Reply:
Fixed, missing namespace added